### PR TITLE
APNSD. temorary workers, with dynamic certificates.

### PR DESCRIPTION
**Pull request by Damirados**

### CODE REVIEW

File: lib/pigeon/apns_worker.ex [hunk 1]
```diff
@@ -6,6 +6,7 @@ defmodule Pigeon.APNSWorker do
   require Logger
 
   @ping_period 600_000 # 10 minutes
+  @inctive_period 60_000 * 60 # 1 hour
```
Comment (svetigrah) [inline lib/pigeon/apns_worker.ex]: inactive

File: lib/pigeon/apns_worker.ex [hunk 2]
```diff
@@ -29,13 +30,18 @@ defmodule Pigeon.APNSWorker do
     mode = config[:mode]
     case connect_socket(config, 0) do
       {:ok, socket} ->
+        state =
```
Comment (svetigrah) [inline lib/pigeon/apns_worker.ex]: What do you do with this state?

Comment (Damirados) [reply lib/pigeon/apns_worker.ex]: garbage line.

File: lib/pigeon/apns_worker.ex [hunk 3]
```diff
@@ -29,13 +30,18 @@ defmodule Pigeon.APNSWorker do
...
+        if config[:dynamic] do
+          time:send_interval(@inctive_period, :terminate_if_passed)
```
Comment (svetigrah) [inline lib/pigeon/apns_worker.ex]: timer?

**Pull request merged**